R/Kelly Wang/kelly-code-review.R

```diff
@@ -0,0 +1,79 @@
+#ticket 20632
+
+library(arcticdatautils)
+library(dataone)
+library(EML)
+library(dplyr)
+library(datapack)
+library(ncdf4)
+
+rm <- "resource_map_urn:uuid:610b0253-5fa1-492b-a6af-abdd012ed325"
+pkg <- get_package(adc, rm)
+doc <- read_eml(getObject(adc, pkg$metadata))
+
+# abstract: a number of the borehole
+doc$dataset$abstract <- gsub("\\(C43-MS1\\)", "borehole with number C43-MS1", doc$dataset$abstract)
+doc$dataset$abstract <- gsub("45 km", "45 kilometers (km)", doc$dataset$abstract)
+doc$dataset$abstract <- gsub("5.2 m", "5.2 meters (m)", doc$dataset$abstract)
```

Comment:
didn't know you could use this to achieve this! I usually use the web editor or `str_replace` but this is a good way too!

Reply:
when I know both I get confused between `gsub` and `str_replace` because the two has the order of the arguments in slightly different positions (pattern first or string first)